presto-main/src/main/java/com/facebook/presto/server/PluginManager.java

```diff
@@ -32,7 +32,6 @@
 import com.facebook.presto.type.TypeRegistry;
```

Comment:
probably this commit should be before the previous one.

Reply:
The order of commits in `git log` is different than in github. I'm not sure why. Do you know how I could fix that?

```
$ git-log | head
2b8fea5fdb Broadcast spatial join
78528482f9 Rewrite a cross join with a spatial filter on top into a spatial join
dd29eaae82 Add benchmark for a broadcast spatial join
32fddb1146 Add support for plugins to LocalQueryRunner
```

Reply:
https://gist.github.com/findepi/7abf17c8a26d3b74e8bb9527caadfdbe

Reply:
@findepi Thank you, Piotr. This workaround worked.